src/commands/init.command.ts

```diff
@@ -155,27 +156,29 @@ const renameGitignoreFiles = async (workspacePath: string) => {
   );
 };
 
-// Needed until https://github.com/unjs/nypm/issues/115 is resolved
-const pmToInstallCommandMap: Record<PackageManagerName, [string, string[]]> = {
-  npm: ['npm', ['ci']],
-  yarn: ['yarn', ['install', '--frozen-lockfile']],
-  bun: ['bun', ['install', '--frozen-lockfile']],
-  pnpm: ['pnpm', ['install', '--frozen-lockfile']],
+type PMTypePromptResult = Pick<PackageManager, 'name'>;
+const promptForPackageManagerSelect = async (): Promise<PMTypePromptResult> => {
+  const temp = await Enquirer.prompt<PMTypePromptResult>({
+    type: 'select',
+    name: 'name' satisfies keyof PMTypePromptResult,
+    message: "Select the package manager you'd like to use:",
+    choices: ['npm', 'bun', 'pnpm', 'yarn'] satisfies PackageManagerName[],
```

Comment:
so `nypm` exports a `packageManagers` list of `PackageManager[]` type. The only issue is, it contains two major versions on `yarn (1.x,3.x)`. Hard defining the list of packages removes the need for filtering out that particular package from displaying in the list of packages. We COULD use the hint property on the choices list items to annotate the version number next to both yarns. 

At the moment doing it this way keeps it consistent with your original intentions, i believe